sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/TelemetryPartB.cs

```diff
@@ -123,6 +129,53 @@ private static void AddPropertiesToTelemetry(IDictionary<string, string> destina
             }
         }
 
+        private static void AddActivityLinksToPartCTags(IEnumerable<ActivityLink> links, ref AzMonList PartCTags)
+        {
+            string msLinks = "_MS.links";
+            // max number of links that can fit in this json formatted string is 107. it is based on assumption that traceid and spanid will be of fixed length.
+            // Keeping max at 100 for now.
+            // todo: define this as constant.
+            int maxLinksAllowed =100;
+
+            if (links != null && links.Any())
+            {
+                var linksJson = new StringBuilder();
+                linksJson.Append('[');
+                foreach (var link in links)
+                {
+                    linksJson
+                        .Append('{')
+                        .Append("\"operation_Id\":")
+                        .Append('\"')
+                        .Append(link.Context.TraceId.ToHexString())
+                        .Append('\"')
+                        .Append(',');
+                    linksJson
+                        .Append("\"id\":")
+                        .Append('\"')
+                        .Append(link.Context.SpanId.ToHexString())
+                        .Append('\"');
+                    linksJson.Append("},");
+
+                    maxLinksAllowed--;
+                    if (maxLinksAllowed == 0)
+                    {
+                        break;
```

Comment:
some logs to indicate we are stopping to look for links, and there are more links.